### PR TITLE
Tweak `Trigger: ` labels

### PR DESCRIPTION
**Pull request by chadwhitacre**
Two changes:

1. **Observe the `Foo: Bar` pattern.** Use the `aliases` feature to [rename](https://github.com/Financial-Times/github-label-sync#label-config-file). "For example, given the following config, GitHub Label Sync will look for labels on GitHub named either 'feature' or 'enhancement' then _update_ them to match the newer config rather than deleting them."

2. **Standardize [**colors**](https://storybook.sentry.dev/?path=/story/core-colors--page).** I recently [introduced](https://github.com/getsentry/sentry/pull/42447) a `Sync: Jira` label, which seems loosely related to `Trigger: `. I propose using the same color for both types.

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
Good stuff, but whatever listens for this trigger will also need updated yes?

File: .github/labels.yml
Comment (evanpurkhiser) [inline .github/labels.yml:12]:
```suggestion
  aliases: ['trigger-getsentry-external']
```

Comment (chadwhitacre) [reply .github/labels.yml]: ty

**Comment by chadwhitacre** [comment]
> whatever listens for this trigger will also need updated yes?

Good call. Putting this in draft while I chase that down.

Any pointers, @asottile-sentry?

**Comment by chadwhitacre** [comment]
https://github.com/search?q=org%3Agetsentry+trigger-getsentry-external&type=code

**Comment by chadwhitacre** [comment]
> whatever listens for this trigger will also need updated yes?

Done in 07adfc83079a9be0318f925132b17157b4cb01c8.

**Comment by chadwhitacre** [comment]
Picked off another in https://github.com/getsentry/develop/pull/795.

I believe that's it (except for [`sentry-test-repo`](https://github.com/getsentry/sentry-test-repo), which is archived).

**Comment by evanpurkhiser** [comment]
Going to let @asottile-sentry approve this one 👍

**Comment by chadwhitacre** [comment]
We should probably pause the automation before landing this, so that the bot can update all the labels without triggering a bunch of unnecessary builds.

**Comment by mattgauntseo-sentry** [comment]
@asottile-sentry this seems like a nice change, would be good to know if you have any concerns.

**Comment by asottile-sentry** [comment]
ah I commented in DMs but not here:

> asottile: I had started doing the same thing but then realized it's going to trigger a bunch of runs and gave up :shrug:
> chad: What if we disable the dispatcher workflow before merging the PR? Then after the label is updated we re-enabled the workflow.
> asottile: that might work -- but it would need to happen some time off hours without disrupting PRs
> chad: Afaict the label automation will rename the label. GH may not even fire a new label event for that, but disabling the workflow should help.
> asottile: ah true
> asottile: hmmmm
> asottile: maybe we can try it in a separate repo and see how it behaves
> asottile: it might also just work

**Comment by chadwhitacre** [comment]
Yeah sorry I've been sitting on this, will try to make another push soon.

**Comment by chadwhitacre** [comment]
> it's going to trigger a bunch of runs

Can we remove the `trigger-getsentry-external` label from [closed PRs](https://github.com/getsentry/sentry/issues?q=label%3Atrigger-getsentry-external+is%3Aclosed)? That would drastically minimize the problem. Right now there's only one [open PR](https://github.com/getsentry/sentry/labels/trigger-getsentry-external), for example, and if we accidentally trigger a new run on one or two PRs I think that's fine?

**Comment by asottile-sentry** [comment]
> > it's going to trigger a bunch of runs
> 
> Can we remove the `trigger-getsentry-external` label from [closed PRs](https://github.com/getsentry/sentry/issues?q=label%3Atrigger-getsentry-external+is%3Aclosed)? That would drastically minimize the problem. Right now there's only one [open PR](https://github.com/getsentry/sentry/labels/trigger-getsentry-external), for example, and if we accidentally trigger a new run on one or two PRs I think that's fine?

yeah that'd probably work

**Comment by chadwhitacre** [comment]
Okay, let's do it! Label cleared. Need an approval here and then I will proceed.

**Review by asottile-sentry — APPROVED** [review]

**Comment by asottile-sentry** [comment]
the whole string needs to be quoted -- otherwise yaml partially parses it as a mapping entity

**Comment by asottile-sentry** [comment]
yeah that should work -- I think `pre-commit run check-yaml --files .github/workflows/*` will show that too iirc

**Comment by chadwhitacre** [comment]
```
$ pre-commit run check-yaml --files .github/workflows/* 
No hook with id `check-yaml` in stage `commit`
$
```

I looked through [`.pre-commit-config.yaml`](https://github.com/getsentry/sentry/blob/174035eddcbee65b2b78be4236d648eff4ac9820/.pre-commit-config.yaml) and don't find `check-yaml`.

**Comment by chadwhitacre** [comment]
Confirmed working on #44551.

https://github.com/getsentry/sentry/actions/runs/4439515190/jobs/7792079708?pr=45511

**Pull request merged**
